### PR TITLE
assignment

### PR DESCRIPTION
**Pull request by alpesh12345**
New pull request, android studio assignment

### CODE REVIEW

File: AndroidLibrary/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#Barcode scanner
+###Simple Usage
+In this app, I had used 
+####Zxing and Lottie library.
```
Comment (samuel-abhishek) [inline AndroidLibrary/README.md:4]: 1. Add the purpose of using this library in your project. 
2. Attach a screenshot or Gif of the application

**Review by samuel-abhishek — CHANGES_REQUESTED** [review]
Please handle the mentioned changes. Also move the library Assignment to from AndroidLibrary/Zxing_Lottie_Library to Android/Alpesh/Zxing_Lottie_Library to avoid merge conflicts for other members.

File: AndroidLibrary/README.md (hunk 1 above)
Comment (alpesh12345) [reply AndroidLibrary/README.md]: All asked changes have been made.

**Review by samuel-abhishek — APPROVED** [review]
LGTM

**Pull request merged**